src/core/pcsxffi.lua

```
@@ -172,6 +173,12 @@ local function jumpToMemory(address, width)
     C.jumpToMemory(address, width)
 end
 
+local function scheduleInterrupt(eCycle)
+    if type(eCycle) ~= 'number' then error 'PCSX.scheduleInterrupt requires a numeric cycle count' end
+    if (eCycle < 0) or (eCycle > 0xffffffff) then error 'PCSX.scheduleInterrupt cycle count parameter should be an unsigned 32-bit integer'
+    C.scheduleInterrupt(eCycle)
```

Luacheck (1.2.0)

[error] expected 'end' (to close 'if' on line 178) near 'C' (indentation-based guess)

(E011)